src/cmap/connection_pool_events.ts

```diff
@@ -29,11 +29,18 @@ export class ConnectionPoolMonitoringEvent {
 export class ConnectionPoolCreatedEvent extends ConnectionPoolMonitoringEvent {
   /** The options used to create this connection pool */
   options?: ConnectionPoolOptions;
+  /** The size of the ConnectionPool's waitQueue */
+  waitQueueSize: number;
 
   /** @internal */
   constructor(pool: ConnectionPool) {
     super(pool);
     this.options = pool.options;
+    this.waitQueueSize = pool.waitQueueSize;
+  }
+
+  get name(): string {
+    return 'ConnectionPoolCreated';
```

Comment:
A `name` getter method has been added to each Connection Pool Event and Command Monitoring Event to function as a discriminator between all the event types to make the `DEFAULT_LOG_TRANSFORM` easier to implement.